content/services/service-1.md

```diff
@@ -1,6 +1,6 @@
 +++
 title = "service1"
-list_description = "loremipsum di loremipsum diloremipsum diloremipsum di loremipsum di"
+list_description = "hello world"
```

Comment:
there is more in the `main` branch for services merge main branch in to this feature branch and you will see the updated services